README.md

```diff
@@ -34,12 +32,12 @@ packages:
 
 To run:
 
-	# start up python
-	> python
+    # start up python
+    > python
         ... (startup messages) ...
-	>>> from pyds9 import *
-	>>> print ds9_targets()
-	>>> d = DS9()
+    >>> from pyds9 import *
+    >>> print ds9_targets()
+    >>> d = DS9()
```

Comment:
@cdeil: this is a good idea. But I would modify first the package structure, so it gets easier to do it.
see #19